portal-web/test/functional/com/liferay/portalweb/tests/enduseree/formsworkflow/metrics/WorkflowMetrics.testcase

```diff
@@ -380,6 +380,135 @@ definition {
 			taskCount = "1");
 	}
 
+	@description = "LRQA-53662 Automate LPS-92053 View process item detail page"
+	@priority = "5"
+	test ViewProcessItemDetailPage {
+		property portal.acceptance = "true";
+
+		ProductMenu.gotoPortlet(
+			category = "Configuration",
+			panel = "Site Administration",
+			portlet = "Workflow");
+
+		Workflow.configureWorkflow(
+			workflowDefinition = "Single Approver",
+			workflowResourceValue = "Blogs Entry");
+
+		JSONBlog.addEntry(
+			entryContent = "Blogs Entry1 Content",
+			entryTitle = "Blogs Entry Title");
+
+		WorkflowMetrics.goToWorkflowProcessMetrics(workflowProcessName = "Single Approver");
+
+		WorkflowMetrics.goToDashboardAllItemsWithFilter(
+			workflowFilterOption = "PENDING",
+			workflowMetricsTab = "DASHBOARD",
+			workflowProcess = "Single Approver");
+
+		AssertElementPresent(locator1 = "WorkflowAllItems#WORKFLOW_INSTANCE_ID");
+
+		var instanceId = selenium.getText("//td[contains(@class,'table-title')]/a[@data-testid='instanceIdLink']/strong");
+
+		echo(" The instanceId is: ${instanceId}");
```

Comment:
Hi @melodywu ,
echo is very helpful when we are creating the script! But, I would prefer to remove them when the scripts are done.